core/players-handbook-2024/classes/class-fighter.xml

```diff
@@ -0,0 +1,337 @@
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_INDOMITABLE</requirements>
+		<description>
+			<p>If you fail a saving throw, you can reroll it with a bonus equal to your Fighter level. You must use the new roll, and you can't use this feature again until you finish a Long Rest.</p>
+			<p class="indent">You can use this feature twice between long rests starting at 13th level and three times between long rests starting at 17th level.</p>
+		</description>
+		<sheet alt="Indomitable" usage="{{indomitable:usage}}/Long Rest">
+			<description>You can reroll a saving throw that you fail with a +{{indomitable:bonus}} bonus. If you do so, you must use the new roll.</description>
+		</sheet>
+		<rules>
+			<stat name="indomitable:usage" value="1" level="9" />
+			<stat name="indomitable:usage" value="1" level="13" />
+			<stat name="indomitable:usage" value="1" level="17" />
+			<stat name="indomitable:bonus" value="level:fighter" />
+		</rules>
+	</element>
+
+	<element name="Level 9: Tactical Master" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_TACTICAL_MASTER">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_TACTICAL_MASTER</requirements>
+		<description>
+			<p>When you attack with a weapon whose mastery property you can use, you can replace that property with the Push, Sap or Slow property for that attack.</p>
+		</description>
+		<sheet alt="Tactical Master">
+			<description>When you attack with a weapon whose mastery property you can use, you can replace that property with the Push, Sap or Slow property for that attack.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 13: Studied Attacks" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_STUDIED_ATTACKS">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_STUDIED_ATTACKS</requirements>
+		<description>
+			<p>You study your opponents and learn from each attack you make. If you make an attack roll against a creature and miss, you have Advantage on your next attack roll against that creaure before the end of your next turn.</p>
+		</description>
+		<sheet alt="Studied Attacks">
+			<description>If you miss with an attack roll, you have Advantage on your next attack roll against that creature before the end of your next turn.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 19: Epic Boon" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_EPIC_BOON">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_EPIC_BOON</requirements>
+		<description>
+			<p>You gain an Epic Boon feat or another feat of your choice for which you qualify. Boon of Combat Prowess is recommended.</p>
+		</description>
+		<sheet display="false">
+		</sheet>
+		<rules>
+			<select type="Feat" name="Epic Boon (Fighter 19)" level="19" />
+
+		</rules>
+	</element>
```

Comment:
```suggestion
		</rules>
	</element>

	<element name="Level 20: Three Extra Attacks" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_THREE_EXTRA_ATTACKS">
		<requirements>!(ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_EXTRA_ATTACK||ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_TWO_EXTRA_ATTACKS||ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_THREE_EXTRA_ATTACKS)</requirements>
		<description>
			<p>You can attack four times instead of once whenever you take the Attack action on your turn.</p>
		</description>
		<sheet display="false" />
		<rules>
			<stat name="extra attack:count" value="4" bonus="extra attack" />
		</rules>
	</element>
```